builder/format.go

```diff
@@ -155,10 +147,12 @@ func (b *Builder) UnstageMixFromBump() error {
 func (b *Builder) CheckBumpNeeded(silent bool) (bool, error) {
 	version, err := b.getLastBuildUpstreamVersion()
 	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
AH ok, so this is where the nil return moves to since you removed it at the top, correct?

Reply:
yep. This is the exception case, where this error is OK. I believe it makes more sense than adding checks for lastver == "" everywhere else.

Reply:
Yeah I agree with this. I'm testing it locally right now myself, but otherwise all the fixups look good to me.